libxrdp/xrdp_sec.c

```diff
@@ -2349,7 +2349,7 @@ xrdp_sec_process_mcs_data_monitors(struct xrdp_sec *self, struct stream *s)
         LOG(LOG_LEVEL_ERROR,
             "[MS-RDPBCGR] Protocol error: TS_UD_CS_MONITOR monitorCount "
             "MUST be less than 16, received: %d", monitorCount);
-        return 1;
+        return 2;
```

Comment:
I'm really t
Suggest macros for these values to tie in the test code with the function. Something like-

```
#define SEC_PROCESS_MONITORS_ERR 1
#define SEC_PROCESS_MONITORS_ERR_TOO_MANY_MONITORS 2
#define SEC_PROCESS_MONITORS_ERR_INVALID_DESKTOP 3
```

An enum is fine too, and will work with the C++ CI.